pages/home.jsx

```diff
@@ -96,11 +96,15 @@ var BlogSection = React.createClass({
       featuredPost: {
         title: "",
         author: "",
-        publishedDate: "",
+        publishedDate: new Date(),
         contentSnippet: "",
-        link: ""
+        link: "https://blog.webmaker.org"
       },
-      latestPosts: []
+      latestPosts: [{
+        title: "",
+        publishedDate: new Date(),
+        link: "https://blog.webmaker.org"
+      }]
     }
```

Comment:
have to set some initial values so we don't run into validation / parsing errors during the initial render